Podfile

```diff
@@ -4,6 +4,10 @@ plugin 'cocoapods-keys', {
   :keys => [
     "GitHubAPIClientId",
     "GitHubAPIClientSecret",
+    "EnterpriseGitHubHostname", # e.g. git.mycompany.com
+    "EnterpriseGitHubAPIPath", # e.g. git.mycompany.com/api/v1"
+    "EnterpriseGitHubAPIClientId",
+    "EnterpriseGitHubAPIClientSecret",
```

Comment:
Here you don't need to add these, because 
- hostname will be parsed from the repo's remote URL
- API path will need to be discovered dynamically
- OAuth won't be supported, so the ClientId and Secret are not necessary.